### PR TITLE
Email registration

### PR DESCRIPTION
**Pull request by rajanibusani**
# Description
Added files for email registration #issue 66 (before it is used to be #issue 40).

Please provide a short summary explaining what this PR is about.

Fixes # (issue)
#66 

# How to test?
npm run storybook, and you can check with giving wrong emails in the input field.
Please provide a short summary how your changes can be tested?

# Checklist

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] This PR is ready to be merged and not breaking any other functionality

### CODE REVIEW

File: src/client/components/EmailRegistration/EmailForm.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+      }
+    }
+  };
+
+  return (
+    <form
+      className="email_form"
+      style={{
+        width: `${formWidth}px`,
+        fontFamily: `${fontFamily}`,
```
Comment (bytewiz) [inline src/client/components/EmailRegistration/EmailForm.js:34]: is there a reason to pass the fontfamily? will it ever change dynamically?

**Review by bytewiz — APPROVED** [review]

File: src/client/components/EmailRegistration/EmailForm.js (hunk 1 above)
Comment (rajanibusani) [reply src/client/components/EmailRegistration/EmailForm.js:34]: @bytewiz, Thank You for your review...!, In the issue 66, they mentioned to use knobs/controls for font family, so i gave that dynamicness, but really not sure not about whether should i do that or not...?

**Comment by ccshah298** [comment]
good job @rajanibusani ...

**Comment by sofiiadidovych** [comment]
Good job, Rajani!

**Review by ccshah298 — APPROVED** [review]

**Review by sofiiadidovych — APPROVED** [review]

**Comment by rajanibusani** [comment]
> Good job, Rajani!

tq sofia

**Comment by rajanibusani** [comment]
> good job @rajanibusani ...

Thank u @ccshah298

**Pull request merged**